setup.sh

```diff
@@ -110,6 +110,21 @@ if [ "$mode" == "production" ]; then
       break
     fi
   done
+
+  while true; do
+  read -rp "Start lasius in https mode (with self-signed certificate)? (y/n) " yn
```

Comment:
"self-signed" could be misleading here. Isn't it a Let's Encrypt certificate? When I read self-signed, I think of a certificate that has not been signed by a trusted CA but one that I have signed myself and therefore is not trusted by browsers by default.